aca_preview/preview.py

```diff
@@ -167,18 +192,90 @@ def get_summary_text(acas):
 
 class ACAReviewTable(ACATable):
     @classmethod
-    def add_review_methods(cls, aca):
+    def add_review_methods(cls, aca, *, obsid=None, loud=False, preview_dir='.'):
         """Add review methods to ``aca`` object *in-place*.
 
         - Change ``aca.__class__`` to ``cls``
         - Add ``context`` and ``messages`` properties.
 
         :param aca: ACATable object, modified in place.
+        :param obsid: obsid (optional)
+        :param loud: print processing status info (default=False)
 
         """
         aca.__class__ = cls
         aca.context = {}  # Jinja2 context for output HTML review
         aca.messages = []  # Warning messages
+        aca.loud = loud
+
+        # Input obsid could be a string repr of a number that might have have
+        # up to 1 decimal point.  This is the case when obsid is taken from the
```

Comment:
Also, I think this "up to 1 decimal point" may be wrong.  For the examples we've seen, sure that .19999 was a poor representation of .2, but it does not seem impossible to me that we'd have 10 or more splits of a single target in a prelim (2 or 3 seems much more likely I agree but seems not worth breaking over more).

Reply:
I set this to 2 decimal points.  I can't immediately find your example of the MATLAB obsid with a poor representation though.

Reply:
You are welcome to the early FEB0419 pickle in ASPECT_ICXC/SFE/2019/FEB0419/FEB0419_Test_proseco.pkl .

Reply:
Passes formatting test, and provided inspiration for the next PR.